setup.py

```diff
@@ -311,7 +311,7 @@ def run_setup(with_extensions):
             'setuptools-scm>1.5.4'
         ],
         install_requires=[
-            'numpy>=1.7',
+            'numpy>=1.7', 'numba==0.53.0rc2'
```

Comment:
Agree with Martin. It would be good to soften this requirement or drop it completely.

Another thing to consider would just be using an optional 3rd party dependency for the algorithm as a whole like ( https://github.com/kiyo-masui/bitshuffle ), which would allow users to get a performant implementation that we can leverage indirectly

Reply:
I had also found this library but didnt know if it would be compatible for _byte_ shuffling, which is what is what appears to be most often applied with netCDF.

Reply:
@martindurant previously suggested raising with the cblosc devs if they would be willing to make their shuffle functions available through a public interface - perhaps that is the best long term approach?